docs/logs/getting-started/LoggerExtensions.cs

```diff
@@ -0,0 +1,32 @@
+// <copyright file="LoggerExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.Logging;
+
+internal static class LoggerExtensions
+{
+    private static readonly Action<ILogger, object, Exception> LogExAction = LoggerMessage.Define<object>(
```

Comment:
lets add a link to the doc explaining the concepts:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/loggermessage?view=aspnetcore-3.1

Reply:
Added.